README.md

```diff
@@ -87,6 +88,19 @@ git checkout -b issue-329
 
 Now, you can suggest contributions, make necessary changes to existing files, or add new files.
 
+## Identifying Documentation Source
+
+Some docs under [open-horizon.github.io/docs](https://github.com/open-horizon/open-horizon.github.io/tree/master/docs) can have its source in different repository.
+
+Use the URL path to identify if the source is [open-horizon.github.io/docs](https://github.com/open-horizon/open-horizon.github.io/tree/master/docs) or another repository. If the source is in different repository then sourced repo name is used `open-horizon.github.io/docs/<SOURCE_REPO_NAME>/docs`.
+
+For example, docs under `open-horizon.github.io/docs/anax/docs` will have URL containing `/docs/anax/docs` (https://open-horizon.github.io/docs/anax/docs/). So it is sourced from `anax` repository (i.e https://github.com/open-horizon/anax/tree/master/docs).
```

Comment:
The prose is longer than necessary.  Let's convert this to a bulleted list.  Intead of examples, explicitly list them all:
- '/docs/anax/docs/' markdown pages are sourced from ....
- '/docs/mgmt-hub/docs/` markdown pages are sourced from ....
- `/docs/kubearmor-integration/docs/` markdown pages are sourced from ...
-

Reply:
addressed